d2l-icon-button-behavior.html

```diff
@@ -0,0 +1,17 @@
+<script>
+	window.D2LBehaviors = window.D2LBehaviors || {};
```

Comment:
So far we've been adding onto `window.D2L.PolymerBehaviors`, so it would be good to be consistent here. Here's [an example](https://github.com/Brightspace/d2l-menu-ui/blob/master/d2l-menu-item-behavior.html#L149). You may also need some comments (see the example) to get polylint to pass.